testing/hyperparams/test_get_set_hyperparams.py

```diff
@@ -181,6 +186,119 @@ def test_step_cloner_should_get_hyperparams_space():
 RAND_INT_META_STEP = RandInt(0, 10)
 
 
+def test_pipeline_should_set_hyperparams():
+    p = Pipeline([
+        SomeStep().set_name('step_1'),
+        SomeStep().set_name('step_2')
```

Comment:
It would be cool to have a section in the documentation page that shows all our unit tests. Just like the example gallery. IDK if this is possible. You can resolve this now to do it later, just throwing an idea here.

Reply:
https://github.com/Neuraxio/Neuraxle/issues/322